subprojects/docs/src/docs/userguide/publishing_overview.adoc

```diff
@@ -13,78 +13,76 @@
 // limitations under the License.
 
 [[publishing_overview]]
-== Publishing
+= Publishing
 
 The vast majority of software projects build something that aims to be consumed in some way. It could be a library that other software projects use or it could be an application for end users. _Publishing_ is the process by which the thing being built is made available to consumers.
 
 In Gradle, that process looks like this:
 
- 1. Define <<publishing_overview:what,what>> to publish
- 2. Define <<publishing_overview:where,where>> to publish it to
- 3. <<publishing_overview:how,Do>> the publishing
+ 1. Define <<#publishing_overview:what,what>> to publish
+ 2. Define <<#publishing_overview:where,where>> to publish it to
+ 3. <<#publishing_overview:how,Do>> the publishing
 
 Each of the these steps is dependent on the type of repository to which you want to publish artifacts. The two most common types are Maven-compatible and Ivy-compatible repositories, or Maven and Ivy repositories for short.
 
-NOTE: Looking for information on upload tasks and the `archives` configuration? See the <<artifact_management>> chapter.
+NOTE: Looking for information on upload tasks and the `archives` configuration? See the <<artifact_management.adoc#artifact_management,Artifact Management>> chapter.
 
-Gradle makes it easy to publish to these types of repository by providing some prepackaged infrastructure in the form of the <<publishing_maven,Maven Publish Plugin>> and the <<publishing_ivy,Ivy Publish Plugin>>. These plugins allow you to configure what to publish and perform the publishing with a minimum of effort.
+Gradle makes it easy to publish to these types of repository by providing some prepackaged infrastructure in the form of the <<publishing_maven.adoc#publishing_maven,Maven Publish Plugin>> and the <<publishing_ivy.adoc#publishing_ivy,Ivy Publish Plugin>>. These plugins allow you to configure what to publish and perform the publishing with a minimum of effort.
 
-++++
-<figure>
-    <title>The publishing process</title>
-    <imageobject>
-        <imagedata fileref="img/publishing-process.png" width="170mm"/>
-    </imageobject>
-</figure>
-++++
+.The publishing process
+image::publishing-process.png[]
 
 Let's take a look at those steps in more detail:
 
 [[publishing_overview:what]]
 What to publish::
-Gradle needs to know what files and information to publish so that consumers can use your project. This is typically a combination of <<glossary:artifact,artifacts>> and metadata that Gradle calls a _<<glossary:publication,publication>>_. Exactly what a publication contains depends on the type of repository it's being published to.
+Gradle needs to know what files and information to publish so that consumers can use your project. This is typically a combination of <<#glossary:artifact,artifacts>> and metadata that Gradle calls a <<#glossary:publication,publication>>. Exactly what a publication contains depends on the type of repository it's being published to.
```

Comment:
Clicking these links also hides their heading below the navbar.

Reply:
There's a CSS bug I thought I fixed that affect all heading links. I'll look into it.